docs/modules/operation/pages/quick-start/notification-config.adoc

```diff
@@ -0,0 +1,63 @@
+
+Here, we will describe how to configure event notifications and path outages.
+Event notifications can be configured to provide UEI-specific coverage, allowing for granular control of incoming information.
+Path outages prevent the generation of excess notifications by identifying network paths where failures may cause multiple nodes to appear offline.
+
+== Configure event notifications
+
+An event notification defines a destination path and message template for a specific event UEI.
+It is possible to define multiple notifications for the same UEI to use different filters, destination paths, and message templates.
+Follow these steps to configure event notifications:
+
+. Click the *Gear* symbol in the top-right of the screen.
+. Under Event Management, click menu:Configure Notifications[Configure Event Notifications].
+A list of events that are configured for notifications is displayed.
+. Click *Add New Event Notification*, choose the UEI that will trigger a notification, and click *Next*.
+** (Optional) Build a rule to match a subset of IP addresses or services, and click menu:Validate Rule Results[Next] or *Skip Results Validation*.
+. In the *Choose Path* screen, specify information to include in the notification, including the destination path, subject (for email messages), body text, and so on.
+. Click *Finish*.
+
+NOTE: Event notification configuration is stored in `$\{OPENNMS_HOME}/etc/notifications.xml`.
+Changes to this file happen immediately, with no need to restart {page-component-title}.
+
+=== Filter rules
+
+A filter rule lets you define which notifications to send or save. Strict rules are always evaluated, and if there is no node or interface associated with the event, the notification will not be saved. 
+
+By default, the filter rule configured in `$\{OPENNMS_HOME}/etc/notifications.xml` (for example, `<rule>IPADDR != '0.0.0.0'</rule>`) does not have the `strict` flag enabled.
+That means if there is an event that is not associated with a node or interface (and therefore no `IPADDR` value), it will bypass validation and the notification would be processed.
+
+A strict rule (for example, `<rule strict="true">IPADDR != '0.0.0.0'</rule>`) will always be evaluated, and if there is no node or interface associated with the event, the notification will not be saved.
+
+== Configure path outages
+
+Configuring a path outage lets you suppress notifications for nodes that appear to be down due to a failure in the network path between the nodes and {page-component-title}.
+For example, if a WAN link fails, all nodes at the remote site served by the WAN link will appear to be down.
+Because you get a notification that the router on the far end of the WAN link is not responding, you don't need notifications for all devices that sit behind that router.
+
+You can configure a path outage for an individual node by setting a parent on the node in a requisition, or create a rule-based configuration for a group of nodes:
```

Comment:
This sentence sounds like we are explaining two options available to the user but then we go on to only list out the steps for one.  I feel like this should be listed as another thing a user can do or it could be explained as beyond the scope of this guide and focus the end user on one path.  It's also possible I am misreading this sentence.

Reply:
Great place to link to a deep dive on how to configure it either way.

Reply:
It looks like we don't have a deep dive on this subject that goes into more detail; I actually lifted these instructions directly from the "Configure path outages" section of the Getting Started page. I think expanding on this in the Deep Dive will need to be a separate ticket; for now, I can remove the second part of the sentence.

```suggestion
You can configure a path outage for an individual node by setting a parent on the node in a requisition:
```